settings/checkstyle/checkstyle.xml

```diff
@@ -10,7 +10,7 @@
 
   <module name="SuppressWarningsFilter" />
   <module name="SuppressionFilter">
-    <property name="file" value="${checkstyleConfigDir}/checkstyle-suppressions.xml"/>
+    <property name="file" value="${config_loc}/checkstyle-suppressions.xml"/>
```

Comment:
So `config_loc` is a built-in property that gets mapped to the directory the configuration file lies in.

Reply:
I believe `configDirectory` is the value injected into `config_loc`

Reply:
Oops, you already mentioned it in the commit description. Sorry about it. 😓